<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalGUI</class>
 <widget class="QMainWindow" name="CalGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>591</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calculation Excercise</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Please set properties.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>260</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Push Next button to start your exercise!</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>290</y>
        <width>261</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="next1">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Next</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>508</width>
        <height>132</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="0" colspan="2">
        <widget class="QCheckBox" name="fraction">
         <property name="text">
          <string>Fraction</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3" colspan="3">
        <widget class="QCheckBox" name="real">
         <property name="text">
          <string>Real</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="oLowRange">
         <property name="minimum">
          <number>-999999</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QSpinBox" name="oUppRange">
         <property name="minimum">
          <number>-999999</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="oNum">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Number of operands</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Number of Questions</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="qNum">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>to</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QSpinBox" name="precisionSpin">
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="precision">
         <property name="text">
          <string>Precision</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Range of operands</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Type of operands:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QCheckBox" name="mulDiv">
         <property name="text">
          <string>*/รท</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3" colspan="3">
        <widget class="QCheckBox" name="power">
         <property name="text">
          <string>power</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Question</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="questionBrowser">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>331</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Your Answer:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="userAnswer">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>310</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OF</string>
      </property>
     </widget>
     <widget class="QLabel" name="presentQ">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>310</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLabel" name="totalQ">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>310</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QPushButton" name="qnext">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>250</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="RestTime">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>110</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="digitCount">
       <number>5</number>
      </property>
      <property name="value" stdset="0">
       <double>20.000000000000000</double>
      </property>
      <property name="intValue" stdset="0">
       <number>20</number>
      </property>
     </widget>
     <widget class="QPushButton" name="StartButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="answer">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="x">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>60</y>
        <width>64</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Result</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>521</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>of</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="5">
        <widget class="QTextBrowser" name="errorBrowser"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_16">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Accuracy</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Correct Questions</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Errors</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" colspan="3">
        <widget class="QLabel" name="resultPercen">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0.00%</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="resultTotal">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="resultCorrect">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>381</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>History</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>CalGUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StartButton</sender>
   <signal>clicked()</signal>
   <receiver>StartButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>CalGUI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StartButton</sender>
   <signal>clicked()</signal>
   <receiver>qnext</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>493</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
